src/core/messageSender.ts

```diff
@@ -19,6 +20,16 @@ class MessageSender {
 
   setChatId(chatId: number) {
     this.chatId = chatId;
+    Preferences.set('general', 'chatId', chatId);
```

Comment:
@kiephil I need to store the chat ID, which is unrelated to any service and will only be read and written in this class. Is `general` the right service key to use?

Reply:
I have not thought about general Preferences but sure, yeah. Do we also need to be able to modify it from the dashboard?

Reply:
No, it's purely internal and not a user preference